go.mod

```diff
@@ -23,4 +26,6 @@ replace (
 	k8s.io/apimachinery => k8s.io/apimachinery v0.19.9
 	k8s.io/client-go => k8s.io/client-go v0.19.9
 	k8s.io/code-generator => k8s.io/code-generator v0.19.9
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.7.2
+	github.com/prometheus/client_golang v1.11.0 => github.com/prometheus/client_golang v1.7.1
```

Comment:
Do we need this [replace](https://github.com/kubeflow/common/blob/a19dc4d55450431b1a91dbdcf793e7c58018bf77/go.mod#L24-L30) or we can remove it since we are using `v0.21.3` version of K8s packages ?

Reply:
I'm trying to keep go packages in common and tf-operator consistent. We can discuss this dependency issue on the next community meeting (August 11th).

Reply:
If we are using `v0.21.3`, shall I get rid of other replace items like `k8s.io/client-go => k8s.io/client-go v0.19.9`?